paddle/platform/dynload/dynamic_loader.cc

```diff
@@ -157,6 +162,14 @@ void GetLapackDsoHandle(void** dso_handle) {
 #endif
 }
 
+void GetNcclDsoHandle(void** dso_handle) {
+#if defined(__APPLE__) || defined(__OSX__)
+  GetDsoHandleFromSearchPath(FLAGS_nccl_dir, "libnccl.dylib", dso_handle);
```

Comment:
In some future PR, we should rename Dso into DSO.

Reply:
Yes, we should use DSO for abbreviation dynamic shared object.